### PR TITLE
Add kubectl kubernetes client wrapper

### PR DESCRIPTION
**Pull request by afbjorklund**
Will connect to a running "k3s" or "k8s" instance.

Similar to other wrappers, but uses conf not sock.

Issue #1300

Needs PR #1301 in order to create the "conf" files

### CODE REVIEW

File: cmd/kubectl.lima [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+: "${LIMA_INSTANCE:=}"
+: "${KUBECTL:=kubectl}"
+
+if [ -z "$LIMA_INSTANCE" ]; then
+  if [ "$(limactl ls -f '{{.Status}}' k3s 2>/dev/null)" = "Running" ]; then
+    LIMA_INSTANCE=k3s
+  elif [ "$(limactl ls -f '{{.Status}}' k8s 2>/dev/null)" = "Running" ]; then
+    LIMA_INSTANCE=k8s
+  else
+    echo "no running instance found, run \`limactl start\` to start an instance" >&2
```
Comment (jandubois) [inline cmd/kubectl.lima:12]: Maybe include a hint that we are only looking for instance names `k3s` and `k8s`, and tell the user to set `LIMA_INSTANCE` if they use any other?

**Review by jandubois — COMMENTED** [review]
Thanks, LGTM, but maybe the error message can be a bit more helpful?

**Comment by afbjorklund** [comment]
It at least outputs the name now, but it doesn't mention which template to use...

```console
$ ./cmd/docker.lima 
instance "docker" does not exist, run `limactl start --name=docker template://docker` to create a new instance
$ ./cmd/podman.lima 
instance "podman" does not exist, run `limactl start --name=podman template://podman` to create a new instance
$ ./cmd/kubectl.lima 
no running instance found, run `limactl start` to start an instance
looked for instance names "k3s" and "k8s", maybe set LIMA_INSTANCE?
```

Maybe should add similar output here, and check if it exists or if it is just stopped...

**Comment by jandubois** [comment]
> Maybe should add similar output here, and check if it exists or if it is just stopped...

Yeah, probably should.

> No `k3s` or `k8s` running instances found. Either start one with
>   limactl start --name=k3s template://k3s
>   limactl start --name=k8s template://k8s
> or set `LIMA_INSTANCE` to the name of your Kubernetes instance.

It quickly gets complicated when you want to check for stopped instances, and then recommand to start those instead of creating a new one, but maybe try to see if you can write it concisely?

**Comment by jandubois** [comment]
Note that the other scripts don't catch it if an instance exists, but hasn't been started. But as you wrote earlier, I think the error message you get in that case should be sufficient already.

**Comment by jandubois** [comment]
I haven't tested again right now, but LGTM. But it requires #1301 to be final before we can approve/merge this, as the name of the `conf` directory may still change.

**Review by jandubois — APPROVED** [review]
Thanks, LGTM

Requires #1301 to be merged

**Comment by AkihiroSuda** [comment]
Next time please set the milestone before merging

**Comment by jandubois** [comment]
> Next time please set the milestone before merging

Sorry about that, I normally do, but I guess I missed it because most of the current PRs already had a milestone assigned, so I forgot to double-check.

**Pull request merged**